### PR TITLE
Gidk 23ghp7 tb stage

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Comment by shalinee-walkover** [comment]
https://app.clickup.com/t/29e7yz - TEST  - TB,PL,BS, eway-bill, setting, sales bifurcation, audit log these modules

File: apps/web-giddh/src/app/tb-pl-bs/components/tb-pl-bs-grid-row.component.html [hunk 1]
```diff
@@ -1,33 +1,92 @@
...
+<ng-container *ngFor="let account of groupDetail.accounts;trackBy: trackByFn">
     <!-- <section class="row row-2 account " [ngClass]="{'isHidden': !account.isVisible }"> -->
-    <section class="row-2 account" *ngIf="account.isVisible">
-        <div class="row"
-            *ngIf="account.name && (account.closingBalance?.amount !== 0 || account.openingBalance?.amount !== 0)">
-            <div class="col-4 account" [ngStyle]="{'padding-left': (padding+20)+'px'}"
-                [innerHTML]="account.name | lowercase | highlight:search"></div>
-            <div class="col-2 account text-right">{{ account.openingBalance?.amount | giddhCurrency }}
-                {{account.openingBalance | recType }}
+    <section class="row-2 account " style="overflow: visible;" *ngIf="account.isVisible || account.isCreated"
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/tb-pl-bs/components/tb-pl-bs-grid-row.component.html:40]: remove inline css @shalinee-walkover

File: apps/web-giddh/src/app/tb-pl-bs/components/tb-pl-bs-grid-row.component.html [hunk 2]
```diff
@@ -1,33 +1,92 @@
-<div class="row row-2 tb-pl-bs-grid-row" [trial-accordion]="groupDetail"
-    *ngIf="groupDetail.groupName && groupDetail.isVisible">
+<div class="row row-2 tb-pl-bs-grid-row" style="overflow: visible;" [trial-accordion]="groupDetail"
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/tb-pl-bs/components/tb-pl-bs-grid-row.component.html:1]: @shalinee-walkover remove inline css

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/shared/amount-field/amount-field.component.ts [hunk 3]
```diff
@@ -16,6 +16,8 @@ export class AmountFieldComponent implements OnInit, OnChanges {
     @Input() public currencyCode: string;
     /* true, ratePrecision value */
     @Input() public ratePrecision: boolean = true;
+    /** this will store giddhCurrency pipe value */
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/shared/amount-field/amount-field.component.ts:19]: Please check if it's used

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/amount-field/amount-field.component.ts:19]: yes it is used

Comment (shalinee-walkover) [reply apps/web-giddh/src/app/shared/amount-field/amount-field.component.ts:19]: Yes @prateek-giddh confirmed, it is used on the subscription page

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**